src/cli/cluster/base_path_proxy.js

```diff
@@ -53,6 +53,28 @@ export default class BasePathProxy {
     server.route({
       method: '*',
       path: `${basePath}/{kbnPath*}`,
+      config: {
+        pre: [
+          (req, reply) => {
+            map(clusterManager.workers, (worker) => {
```

Comment:
Is there an unwritten rule saying we should write single-argument arrow functions without parenthesis?